packages/framer-motion/src/animation/legacy-popmotion/inertia.ts

```diff
@@ -62,9 +61,9 @@ export function inertia({
         })
     }
 
-    if (isOutOfBounds(from)) {
+    if (isOutOfBounds(origin)) {
         // Start the animation with spring if outside the defined boundaries
-        startSpring({ from, velocity, to: boundaryNearest(from) })
+        startSpring({ velocity, keyframes: [origin, boundaryNearest(origin)] })
```

Comment:
What does `boundaryNearest` mean?

Reply:
Renamed to `findNearestBoundary` - this is ultimately where drag constraints get applied, that kind of animation